test/fixtures/env-config/browserslist

```diff
@@ -6,6 +6,9 @@ ie 10
 ff 50
 chrome 55
 
-[production]
+[staging]
+edge 16
```

Comment:
This line means merging?

Reply:
@ai yes, this is rare use case, but it is sometimes need

Reply:
From Ruby on Rails I had big experience in YAML. And all that flexibility to reduce few lines in config made bad things with the format.

So after YAML story, I think Browserslist config should be very simple. Let’s throw a error in this case. One header will reduce readability (it is easy to miss second section below).

What do you think?

Reply:
@ai throw error if env section already define?

Reply:
@evilebottnawi yeap

Reply:
@ai not throw?

Reply:
I will fix it (plan to release 3.0 today)

Reply:
@ai :+1: